core/trino-main/src/test/java/io/trino/execution/TestNodeScheduler.java

```diff
@@ -785,7 +786,7 @@ public void testEmptyAssignmentWithFullNodes()
             TaskId taskId = new TaskId(new StageId("test", 1), task, 0);
             task++;
             MockRemoteTaskFactory.MockRemoteTask remoteTask = remoteTaskFactory.createTableScanTask(taskId, node, ImmutableList.copyOf(assignments1.get(node)), nodeTaskMap.createPartitionedSplitCountTracker(node, taskId));
-            remoteTask.startSplits(20);
+            remoteTask.startSplits(15); // 15 running, 5 queued, no maxSplitWeightPerNode adjustment
```

Comment:
why do we need to lower it?

Reply:
If we set all task as running - queue will be extended.